### ISSUE TITLE
A11y: Step by step mode -  can select invisible fields

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5063]
Using a screenreader in Safari it is possible to select fields that should not yet be active and available in step-by-step mode. Elements can be selected and will be read out by screenreader, but cannot be interacted with.

#Safari

!Screen Recording 2024-10-09 at 18.01.23.mov|width=1538,height=1306,alt="Screen Recording 2024-10-09 at 18.01.23.mov"!

#Chrome

In Chrome, it is impossible to navigate even the first option via keyboard by pressing {{Tab}}

The product has 14 steps to complete which cannot be reached via keyboard.

!Screen Shot 2024-10-14 at 09.44.32 AM.png|width=2368,height=1314,alt="Screen Shot 2024-10-14 at 09.44.32 AM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-740**



[created from CUS-740, comment 502172]
